ssc/Pos/Ssc/Extra/Logic.hs

```diff
@@ -235,9 +235,6 @@ sscApplyBlocksFinish
     => SscGlobalState ssc -> m [SomeBatchOp]
 sscApplyBlocksFinish gs = do
     sscRunGlobalUpdate (put gs)
-    inAssertMode $
-        logDebug $
-        sformat ("After applying blocks SSC global state is:\n"%build) gs
```

Comment:
Ok I've restored it, and made sure that `scripts/bench/buildbench.sh` will set `-asserts`.